_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
 Ultimately, he took over the responsibility for this PR. He updated and polished the UI, fixed bugs, got everyone and their grandfather to test the changes, and pulled us to the finish line!
 
 ## Improvements
 
-We are not done yet with what this release brings you (you may want to take a few sips of that beverage now). If you use an environment where resizing an app is possible, NewPipe can do this now as well, thanks to [@TobiGr](https://github.com/TobiGr).
+Take this moment to sip from the nectars of your beverage. [@TobiGr](https://github.com/TobiGr) lets you resize NewPipe provided it is possible in your environment.
 
-Have you ever shared a URL to NewPipe, just to be greeted with a little "Unsupported URL" toast? Well, the app still doesn't support those kinds of URLs with this release, but at least we show you a way better error message now! It allows you to reshare the URL or just offers to open it in a browser instead. All this, thanks to [@Stypox](https://github.com/Stypox) and to [@webber-naut](https://github.com/webber-naut) for bringing our attention to this in the first place. 
+Have you ever been sandwiched between sharing a URL to NewPipe, and a little "Unsupported URL" toast? Well, those URLs are still unsupported, but now the error message is great! It allows you to reshare the URL or just open it in a web-browser instead. All this, thanks to [@Stypox](https://github.com/Stypox) and [@webber-naut](https://github.com/webber-naut) for bringing this to our attention in the first place. 
 
-Talking about sharing, when you share a live stream with someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94). Thanks.
+Talking about sharing, when you share a live stream with someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94). Thanks :)
 
-The header layout has been improved: the alignment on the sides is consistent now and longer headings can fit themselves on two lines instead of just one.
+The header layout has improved: The alignment on the sides is consistent, and longer headings can expand to two lines.
 
-If you ever tried to search something and then lost connection to the internet, or just tried to search while being offline, you would have noticed that NewPipe didn't let you type your search term. Instead it kept on rudely interrupting you with a network error. That is fixed now. You can type all you want, and it will only fail if you actually hit enter or press the search button. Both of these changes were brought to you by [@Stypox](https://github.com/Stypox). Thanks!
```

Comment:
Another ride. Great. Why does anyone have to be in it. So long.

Reply:
> So long.

"If you tried X or Y, Z happened. This is fixed now. Now you can do Y freely." Doesn't seem all that problematic.

> "Internet" with big i.

True.